src/docs/clustering/distributed-data.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: docs.hbs
+title: Akka distributed data
+---
+# Distributed data
+
+Akka.DistributedData plugin can be used as in-memory, highly-available, distributed key-value store, where values conform to so called **Conflict-Free Replicated Data Types** (CRDT). Those data types can have replicas across multiple nodes in the cluster, where DistributedData plugin has been initialized. We are free to perform concurrent updates on replicas with the same corresponding key without need of coordination (distributed locks or transactions) - all state changes will eventually converge with conflicts being automatically resolved, thanks to the nature of CRDTs. To use distributed data plugin, simply install it via NuGet:
```

Comment:
@Horusiath I'd include a link here to some reference material on CRDTs for readers who are interested in learning more.